packages/expo-modules-core/ios/Swift/Networks/ExpoRequestLoggerProtocol.swift

```diff
@@ -0,0 +1,183 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import Foundation
+
+/**
+ A `URLSession` interceptor which passes network events to its delegate
+ */
+@objc
```

Comment:
We may need to explicitly set an Objective-C name (with `EX` or `Expo` prefix) so it gets versioned correctly